tests/integration/test_remote_transfer.py

```
@@ -205,65 +219,96 @@ async def mock_post(url, headers, content, timeout):
 
-            mock_response_receive = MagicMock(status_code=200, raise_for_status=lambda: None, json=lambda: {"status": "success"})
-            mock_client.post.return_value = mock_response_receive # Mock client for the actual file transfer
+            mock_response_receive = MagicMock(
+                status_code=200, raise_for_status=lambda: None, json=lambda: {"status": "success"}
+            )
+            mock_client.post.return_value = (
+                mock_response_receive  # Mock client for the actual file transfer
+            )
 
             # Simulate remote sending chunks back
             async with aiofiles.open(remote_temp_file, "rb") as f:
```

Ruff (0.15.1)

[error] Undefined name `aiofiles`

(F821)